doc/py_tutorials/py_setup/py_setup_in_windows/py_setup_in_windows.markdown

```diff
@@ -44,6 +67,21 @@ Installing OpenCV from prebuilt binaries
 If the results are printed out without any errors, congratulations !!! You have installed
 OpenCV-Python successfully.
 
+Installing OpenCV in Anaconda
+-------------------------------
+
+To install OpenCV in your Anaconda environment perform the following steps:
+
+-#  Download and install anaconda environment Python from [here](https://www.anaconda.com/download)
+-#  Open Anaconda Prompt
+
+-#   Use this command to install opencv in Anaconda
+      @code{.sh}
+      conda install -c conda-forge opencv
+      @endcode
+-#   Press Enter
+-#   Press 'y' when prompted and press Enter. This completes the installation.
```

Comment:
Remove this step and modify the step above: `conda install -y -c conda-forge opencv`

Reply:
@ParamThakkar123